content/how_to_guide.md

```diff
@@ -172,47 +175,63 @@ This drop-down menu provides links to other pages with more water data or descri
 ## Frequently Asked Questions
-### Are real-time pages of today being discontinued? 
+
+### 1. I think the old website works fine, why do you feel the need to change it? 
+The old website has served the public well, but it is 25 years old—that’s about 80% of the entire history of the World Wide Web!  Over the last 10 years, we have observed declining user satisfaction with the old website; novice users in particular tell us the legacy pages are challenging to understand and not intuitive to use.  Under the hood, the programming has become outdated and has become difficult to maintain and it is practically impossible to add new features.  The best way for us to serve the public well into the future is to modernize our pages. 
+
+### 2. Will you be adding the ability to show data from multiple monitoring locations at the same time? 
+Not yes, not no... we need more information first, so we are currently gathering use-cases to better understand our users’ needs. We plan to undertake a “discovery” research project on these needs in fiscal year 2022. These kinds of reports that summarize multiple monitoring locations are valuable for certain advanced users, and we need to ensure all members of the public can derive value from these reports. 
+
+### 3. What will happen to all of the other USGS water data on legacy pages? I don’t see it on the new pages... 
+The NextGen pages will continue to evolve over time, adding new features and incorporating key information available on legacy pages. Current-condition real-time water data account for over 90% of our water-data web page requests, but we understand that other types of water data are very important to some users. These other forms of water data available in legacy pages—daily values, statewide streamflow tables, annual-peak values, and many more—will be evaluated for modernization in the coming years.  Eventually the entire old legacy website will be turned off (decommissioned) but that will not happen until we are able to evaluate the importance of all water data currently available on legacy pages. 
+
+### 4. Why do you now show only one hydrograph instead of a long page of many hydrographs? 
+<b>Interactive:</b> Users, #WeHearYou – you wanted an interactive hydrograph, so we made that a top priority in the early development of the new pages. Users wanted to view the data on their mobile devices, and they wanted to interact with many elements of the graph like data shown, time span, and calculated median data. 
+<b>Fast load time:</b> A single interactive hydrograph is fast for the default 7-day view, but takes some time to load data as users select longer time spans. Based on current technology, we do not advise that a single page provide interactive hydrographs for all possible parameters, as it would not allow for fast loading on mobile devices. Although there are multiple graphs on the legacy pages, those are not interactive, and the technology difference is vast between the two styles of use.
```

Comment:
I am not sure what your intention is here but the way this is coded will be in one big paragraph. If your intention is for each <b></b> tag starts a new line, then you will have to put a blank line in between or use a markdown list (which you can do be adding a "*" in front of the <b>